port/unix/omrsysinfo.c

```diff
@@ -78,6 +78,8 @@
 #include <nl_types.h>
 #include <langinfo.h>
 
+#include <ctype.h>
+#include <stdint.h>
```

Comment:
* `ctype.h` is not needed (we can use just `strtoull()` without `isdigit()`)
* `stdint.h` is already included via `inttypes.h`